include/color_spinor_field_order.h

```diff
@@ -1023,19 +1026,20 @@ namespace quda
       {
         real v[length_ghost];
         norm_type nrm
-          = isFixed<Float>::value ? vector_load<float>(ghost_norm[2 * dim + dir], parity * faceVolumeCB[dim] + x) : 0.0;
+          = isFixed<Float>::value ? vector_load<float, 1>(ghost_norm[2 * dim + dir], parity * faceVolumeCB[dim] + x)[0] : 0.0;
+        norm_type nrm_shift = -nrm * 12582912.0f;
```

Comment:
This assumes the scale type will always be `fp32`, which is reasonable, but we might want to make it some type of `scale` trait or constexpr function or whatnot?